gaea/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
```

Comment:
Top comment here (adding `"noImplicitThis"`) still needs to be done to pass tests.